### ISSUE TITLE
Grid '0' to hide the widget at certain size does not work

### ISSUE THREAD

**Issue by devsportties** [mohamed-selim-a/ResponsiveGrid_Flutter#18]
I want to set the value '0' to the size 'xs', however, it still appears on the screen.

**Comment by benji-farquhar** [comment]
Yeah, this package is only very usable with this feature added in. The most basic and everyday use case is, say, you want to have a column full width on "xs" and "sm" size, then on "md" size, you want half width and a new column two half-width. But that means you need column two to be hidden on"xs" and "sm", and you can't do it.

**Comment by muhammad369** [comment]
Hi there

Excuse me for my 2-years-late response
I've just added this feature to the version that I'll upload soon

By the way, there are many ways to use the package, not necessarily by utilizing the `ResponsiveGridRow`
for your case you can use the `responsiveValue()` function and set width to different values (may be zero) for each size tier

It's also possible to put your widget inside a Visibility widget, and set different values for the visible property to each size tier , through the responsiveValue function
